image/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM quay.io/fedora/fedora-minimal:36
 
-RUN microdnf install -y make git golang python3-pip tar && microdnf clean all -y
+RUN microdnf install -y make git golang python3-pip tar xz && microdnf clean all -y
```

Comment:
Better install shellcheck with dnf, keep the case sensitive pkg name in mind (`ShellCheck`)